src/main/java/org/broadinstitute/consent/http/db/mapper/DataAccessRequestMapper.java

```diff
@@ -25,7 +25,7 @@ public DataAccessRequest map(ResultSet resultSet, StatementContext statementCont
     if (hasColumn(resultSet, "dar_code")) {
       dar.setDarCode(resultSet.getString("dar_code"));
     }
-    dar.setParentId(resultSet.getString("parent_id"));
+    dar.setParentId(resultSet.getInt("parent_id"));
```

Comment:
We were coercing the `bigint` value in the database into a `String`.  I expect we'll go back to `String` in a different pull request, but this change makes our consent and duos-ui code clear on what the value is today.